server/src/main/resources/resourceBundles/messages_en.properties

```diff
@@ -11,6 +11,11 @@ CobolVisitor.misspelledWord=A misspelled word, maybe you want to put %s
 CobolVisitor.paragraphNotDefined=The following paragraph is not defined: %s
 CobolVisitor.progIDIssueMsg=There is an issue with PROGRAM-ID paragraph
 CobolVisitor.subroutineNotFound=%s: Subroutine not found
+CobolVisitor.theSameGroup=Redefined element should belong to the same group: %s
+semantics.redefinedContainValue=Redefined clause cannot contain VALUE: %s
+semantics.redefineRedefined=Redefined variable cannot be redefined: %s
```

Comment:
redefined as?

Reply:
Let's wait for technical writer :-) I am not sure what to mention here